lib/import/helpers/brca/providers/rj1/rj1_common_methods_processor.rb

```diff
@@ -0,0 +1,195 @@
+module Import
+  module Helpers
+    module Brca
+      module Providers
+        module Rj1
+          # Processor for common exctraction methods for FullScreen, Targeted, Ashkenazi and Polish
+          module Rj1CommonMethodsProcessor
+            include Import::Helpers::Brca::Providers::Rj1::Rj1Constants
+
+            #####################################################################################
+            ################ HERE ARE COMMON METHODS ############################################
+            #####################################################################################
+
+            # rubocop:disable Metrics/CyclomaticComplexity
+            def no_cdna_variant?
+              return if @brca1_mlpa_result.nil? && @brca2_mlpa_result.nil?
```

Comment:
Please see previous comment about return values of predicate methods

Reply:
Hi @ollietulloch, predicament methods now return false rather than nil